src/routes/todo.routes/getToDos.ts

```diff
@@ -1,13 +1,12 @@
 import { Router, Request, Response } from "express";
-import getToDos from "../../repository/todo.repository/getToDos";
+import getToDos from "../../repository/todo.repository/todoRepository";
 
-export default function getToDosItems(): Router {
-  const getToDosRouter = Router();
+export function todoRouter(): Router {
+  const todoRouter = Router();
 
-  getToDosRouter.get("/all", async (req: Request, res: Response) => {
-    const todos: [] = (await getToDos()) as [];
+  return todoRouter.get("/all", async (req: Request, res: Response) => {
```

Comment:
No need for `all` word inside URL. When you have a `GET` endpoint, without the `:id` parameter, in REST that means you're fetching ALL entities.